carbon-design-system/carbon
# fix(flexgrid): include flexgrid styles in `@carbon/styles` and `@carbon/react`

FlexGrid styles weren't being output from the new packages. I also had to move aspect ratio to it's own module to avoid duplicating it in both the css-grid and flex-grid styles

#### Changelog

**New**

- include flexgrid styles in `@carbon/styles` and `@carbon/react`
- add flexgrid stories to v11 storybook

**Changed**

- move aspect ratio out of grid to it's own module within `@carbon/styles`

#### Testing / Reviewing

- New v11 storybook should show FlexGrid stories rendered properly with styles.
- Aspect ratio stories in both v10 and v11 storybook should be unaffected
- To test 12 column configuration:
  - View the FlexGrid "Responsive" story in the large breakpoint - the columns do not take up the full space because there is 16 columns now by default.
    - ![image](https://user-images.githubusercontent.com/3360588/153303959-39af36e8-0b41-4810-b91d-2073500ad79a.png)

  - Modify storybook's `@use` to configure the new `$flex-grid-columns` option
  https://github.com/carbon-design-system/carbon/blob/83f414f2c9186796bd31959786369c903eb959af/packages/carbon-react/.storybook/styles.scss#L7-L8
   ```diff
   - @use '../index.scss' as styles
   + @use '../index.scss' as styles with (
   +   $flex-grid-columns: 12
   + );
   ```
  - View the FlexGrid "Responsive" story in the large breakpoint again - the columns now take up the full space because 12 columns have been configured
    - ![image](https://user-images.githubusercontent.com/3360588/153304081-c8681909-5090-4d96-97b1-7ed1064708e4.png)
